events/syscall/polkit_local_privilege_escalation_vulnerability.go

```diff
@@ -0,0 +1,32 @@
+//go:build linux
+// +build linux
+
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	PolkitLocalPrivilegeEscalationVulnerability,
+	events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func PolkitLocalPrivilegeEscalationVulnerability(h events.Helper) error {
```

Comment:
```suggestion
func PolkitLocalPrivilegeEscalationVulnerability(h events.Helper) error {
```

This won't match the exact rule name, which is [Polkit Local Privilege Escalation Vulnerability (CVE-2021-4034)](https://github.com/falcosecurity/rules/blob/c0a9bf17d5451340ab8a497efae1b8a8bd95adcb/rules/falco-sandbox_rules.yaml#L1657)

I guess the correct name should be `PolkitLocalPrivilegeEscalationVulnerabilityCVE20214034,` but I have to double-check.

This convention is important because the [tester](https://github.com/falcosecurity/event-generator/blob/298059d68236a7555f52d462b54c140868d29b34/pkg/tester/tester.go#L91) package relies on that to match rules names:

https://github.com/falcosecurity/event-generator/blob/298059d68236a7555f52d462b54c140868d29b34/events/registry.go#L87-L96